works around this crash frequently happening in some situations,
../src/freedreno/vulkan/tu_knl_kgsl.cc:572:
VkResult wait_timestamp_safe(int, unsigned int, unsigned int, uint64_t):
Assertion errno == ETIMEDOUT' failed.
by allowing the program to proceed and printing a warning instead, which can sometimes work
and preserve the GPU connection.

--- a/src/freedreno/vulkan/tu_knl_kgsl.cc
+++ b/src/freedreno/vulkan/tu_knl_kgsl.cc
@@ -579,7 +579,8 @@ wait_timestamp_safe(int fd,
 
          wait.timeout = timeout_ms;
       } else if (ret == -1) {
-         assert(errno == ETIMEDOUT);
+         mesa_logw("tu_knl_kgsl.cc wait_timestamp_safe() errno: %s\n", strerror(errno));
+         //assert(errno == ETIMEDOUT);
          return VK_TIMEOUT;
       } else {
          return VK_SUCCESS;
